README.md

```diff
@@ -60,3 +60,25 @@ parameters:
                 - 2     # /Home
                 - 43    # /Media
 ```
+
+## Default location for content type
+If suggested locations are given for content type (or default) the first location in the list will be preselected as location.
+
+## Preselected content type
+To set the content type you have to provide 'contentTypeIdentifier' in the config of the 'contentDiscover' event.
```

Comment:
To set the Content Type you have to provide the 'contentTypeIdentifier' in the config of the 'contentDiscover' event.